web/static/web/css/header.css

```diff
@@ -1,144 +1,274 @@
+
+    margin: 0;
+    justify-self: end;
+    display: grid;
+    grid-template-columns: var(--estimated-max-name-length) calc(var(--logo-padding) + 10px);
+    justify-items: end;
+    font-size: var(--font-size);
+}
+
+#side-nav > .item {
+    margin-right: 0;
+}
+
+#nav-user-dropdown .menu {
+    background-color: rgb(34, 43, 52);
+    border-radius: 7px;
+    /* For some reason doesn't work with `border: 2px rgb(220, 221, 222)` */
+    border: 2px solid rgb(220, 221, 222);
+}
+
+#nav-user-dropdown .menu > .item {
+    /* Fixes weird centering of dropdown items, caused by Semantic UI setting dropdown menu to `display: flex !important`
+       when it's too close to the right window edge */
+    align-self: auto;
+}
+
+#nav-user-dropdown .menu > .item:hover {
+    background-color: rgba(255, 255, 255, .08) !important; /* Overrides Semantic UI */
+}
+
+#side-nav .lang-link {
+    position: absolute;
+    margin: 0;
+    top: 0;
+    right: 0;
+}
+
+#side-nav .lang-link .link {
+    margin: 0;
+    align-self: start;
+}
+
+#side-nav .lang-link .link .text {
+    margin-right: 0.5em;
+}
+
+#side-nav .lang-link .link .globe.icon {
+    margin: 0;
+}
+
+/* Narrower space between elements in navbar */
+@media screen and (max-width: 1100px) {
+    #header {
+        --logo-padding: 1em;
+        --nav-space-between: 1em;
     }
 }
 
-@media screen and (max-width: 730px) {
-    .limg.large {
-        display: inline;
+/* Switch to mobile layout with burger */
+@media screen and (max-width: 992px) {
+    #header {
+        --burger-space: minmax(calc(4em + var(--burger-margin)), 1fr);
+
+        grid-template-columns: var(--burger-space) auto var(--burger-space);
+        grid-template-rows: var(--header-height);
     }
 
-    .limg.small {
-        display: none;
+    #header.active {
+        height: auto;
     }
 
-    .limg.medium {
-        display: none;
+    #header div.logo-centering {
+        display: inline;
     }
 
-    .nav {
-        grid-template-columns: 1fr 3fr;
+    #header .logo {
+        grid-template-columns: 1em var(--logo-space) 1em;
     }
 
-    .nav .logo {
-        margin-right: 80px;
+    #burger {
+        display: inline;
     }
 
-    .nav .menu {
-        display: none;
-        grid-row-start: 2;
+    #nav {
+        grid-row: 2;
         grid-column-start: 1;
-        grid-column-end: 3;
+        grid-column-end: 4;
+        margin: 0;
+        justify-self: center; /* Makes the navbar not take up the entire width of its parent, which most noticably applies to the buttons inside */
+    }
+
+    #nav,
+    #side-nav {
+        display: none;
     }
 
-    .nav.active .menu {
+    #header.active #nav,
+    #header.active #side-nav {
         display: inline;
-        padding-left: 10px;
-        padding-bottom: 20px !important;
+    }
 
+    #nav > .item,
+    #side-nav > .item {
+        margin: 0;
     }
 
-    .nav.active {
-        height: auto;
+    #side-nav {
+        align-self: end;
```

Comment:
Does not seem to have an effect here, as this is the only content in this row of the grid.

Reply:
True, I think I kept it there to signal that `#side-nav` in principle should stick to the bottom of the expanded burger menu, if the header for some reason got its height increased, or something.. I'd like to keep it there because of that, but might as well remove it :)

Reply:
Probably just as good to keep it there.